File: public/programs/java/p13.java
import java.util.*;
class p13
{
	public static void main(String[] args) {
		int n,f1=0,f2=1;
		Scanner scan=new Scanner(System.in);
		System.out.print("Enter n value: ");
		n=scan.nextInt();
		System.out.print("The fibonacci series is ");
		for(int i=1;i<=n;i++)
		{
			System.out.print(f1+" ");
			int sum=f1+f2;
			f1=f2;
			f2=sum;
		}
	}
}

File: public/programs/cpp/p13.cpp
// Fibonacci series

#include<iostream.h>
using namespace std;

int main()
{
	int n,i;
	int n1,n2,n3;
	cout<<"Enter a number:";
	cin>>n;
	
	n1=0;
	n2=1;
	cout<<n1<<n2;
	for(i=3;i<=n;i++)
	{
		n3=n1+n2;
		cout<<n3<<" ";
		n1=n2;
		n2=n3;
		
	}
}
		
		